### ISSUE TITLE
Процент доступных в списке проверок

### ISSUE THREAD

**Issue by WST** [orgtechservice/roskombox#11]
Алексей запрашивает, чтобы был процент доступных ссылок в списке проверок, я с ним соглашусь, стоит сделать…

**Comment by dserov** [comment]
Добрый день!
Есть нюанс. Некоторые ссылки ресолвятся в 127.0.0.1

```
C:\>nslookup pornofaza.tv 8.8.8.8
╤хЁтхЁ:  google-public-dns-a.google.com
Address:  8.8.8.8

Не заслуживающий доверия ответ:
╚ь :     pornofaza.tv
Address:  127.0.0.1
```

следовательно, при проверке с машины с веб-кой роскомбокс, вместо сайта будет открываться веб-ка и ошибочно будет считаться, что сайт не заблокирован. Надо считать такие ссылки, как неоткрывшиеся, наверное. 
Поразила скорость проверки! 6 минут и готово!

**Comment by WST** [comment]
См строки 114-115 и 249-250 файла `portal/management/commands/roskom_check.py` — такая ситуация учитывается. Мы тоже провайдер и тоже не заинтересованы в ложных срабатываниях (правда, используем не Роскомбокс, а другие скрипты, интегрированные с нашей корпоративной админкой).

Если сайт, всё-таки, отобразился как незаблокированный, то, скорее всего, там соединение прошло по IPv6, а не IPv4. Опять же, гляну как только смогу, ориентировочно завтра.

**Comment by dserov** [comment]
Да, строки смотрел. Может имеет смысл reply дописывать в таблицу available_links для наглядности ответа и код ответа сервера. IPv6 не используется. Тередо занулен на роутере. Сделал print для дебага и хотел запустить из командной строки roskom_check.py, но не справился. Подскажите, как правильно это сделать. К сожалению, не владею питоном, виртуаленвами и джанго.

**Comment by WST** [comment]
Вы имеете в виду сохранять ответ в базу данных? У нас начальство не очень-то это одобряет (в наших корп.проверках изначально так было, но потом поступило распоряжение убрать). Сейчас спросил разрешение, дали добро сделать это опцией, так что сделаю. На будущее если у вас возникает потребность в новой функциональности, создавайте отдельный тикет, чтобы не создавать путаницу, этот тикет исключительно про отображение процента доступных URL.
